packages/graphile-build-pg/src/QueryBuilder.js

```diff
@@ -874,6 +907,32 @@ order by (row_number() over (partition by 1)) desc`;
     this.lock("selectCursor");
     this.lock("select");
   }
+  buildChild() {
+    const options = { supportsJSONB: this.supportsJSONB };
+    const child = new QueryBuilder(options, this.context, this.rootValue);
+    child.parentQueryBuilder = this;
+    return child;
+  }
+  buildNamedChildFrom(
```

Comment:
Given this enters into a new special mode, I think the name needs to reflect this. I'm not sure what a good name would be though, but this one doesn't seem quite right. Maybe:

```
  buildNamedSingleFieldChild(
```

what do you think?

---

Thinking about it more, I'm not sure it should be limited to a single field; for example the following should be valid:

```sql
select * from a where (a1, a2) in (select b1, b2 from b)
```

So maybe it's more that it has a fixed select clause:

```
buildNamedChildSelecting(name, expression, from, alias?)
```